src/Microsoft.Azure.SignalR.Management/IInternalServiceHubContext.cs

```diff
@@ -11,29 +11,20 @@
 
 namespace Microsoft.Azure.SignalR.Management
 {
-    public interface IServiceContext : IDisposable
+    internal interface IInternalServiceHubContext : IServiceHubContext
```

Comment:
you keep it here, still has a plan to use it in the future?

Reply:
Actually it is a new file. It is different from `IServiceContext`. Use it to avoid add new internal methods to `IServiceHubContext` and disable users to implement `IServiceHubContext` for test. Some methods might be moved to `IServiceHubContext` and made public if ready.

Reply:
why IInternalService**Hub**Context  has method `NegotiateAsync` and `GetServiceEndpoints`? what's the plan?